baseplate/__init__.py

```diff
@@ -224,41 +249,30 @@ def configure_metrics(self, metrics_client):
         self._metrics_client = metrics_client
         self.register(MetricsBaseplateObserver(metrics_client))
 
-    def configure_tracing(self, tracing_client, *args, **kwargs):
+    def configure_tracing(
+        self, tracing_client: "baseplate.observers.tracing.TracingClient"
+    ) -> None:
         """Collect and send span information for request tracing.
 
         When configured, this will send tracing information automatically
         collected by Baseplate to the configured distributed tracing service.
 
-        :param baseplate.observers.tracing.TracingClient tracing_client: Tracing
-            client to send request traces to.
+        :param tracing_client: Tracing client to send request traces to.
 
         """
         # pylint: disable=cyclic-import
-        from baseplate.observers.tracing import make_client, TraceBaseplateObserver, TracingClient
-
-        # the first parameter was service_name before, so if it's not a client
```

Comment:
Dropped support for the deprecated way of setting up trace observers to make typing simpler here. This whole method shouldn't really get seen much now with `configure_observers()`.